scripts/ci/selective_ci_checks.sh

```diff
@@ -475,8 +475,11 @@ function get_count_kubernetes_files() {
     echo
     local pattern_array=(
         "^airflow/kubernetes"
+        "^airflow/executors/kubernetes_executor.py"
         "^chart"
-        "^tests/kubernetes_tests"
+        "^tests/kubernetes"
```

Comment:
This should not be there. Those "kubernetes_tests" are special tests that are not the same as all the "unit tests". They do not need CI image, instead they are using local virtual env to run tests and Kind-Kubernetes started on the host using the PROD image. 
```suggestion
```